### ISSUE TITLE
Multiple outputs functionality is broken

### ISSUE THREAD

**Issue by serzhshakur** [cucumber-rs/cucumber#231]
With [retrying mechanism](https://github.com/cucumber-rs/cucumber/commit/c1d919a476ebd8f87633e49d5ca2f8b647a0fce7) being introduced it is no longer possible to execute tests with [multiple outputs](https://cucumber-rs.github.io/cucumber/main/output/multiple.html).

The error is thrown by clap when trying to execute tests

```
thread 'main' panicked at 'Command cucumber: Argument names must be unique, but 'verbose' is in use by more than one argument or group', /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/builder/debug_asserts.rs:95:13
stack backtrace:
   0: rust_begin_unwind
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/core/src/panicking.rs:142:14
   2: clap::builder::debug_asserts::assert_app
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/builder/debug_asserts.rs:95:13
   3: clap::builder::command::Command::_build_self
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/builder/command.rs:3918:13
   4: clap::builder::command::Command::_do_parse
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/builder/command.rs:3788:9
   5: clap::builder::command::Command::try_get_matches_from_mut
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/builder/command.rs:708:9
   6: clap::builder::command::Command::get_matches_from
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/builder/command.rs:578:9
   7: clap::builder::command::Command::get_matches
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/builder/command.rs:490:9
   8: clap::derive::Parser::parse
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.13/src/derive.rs:82:27
   9: cucumber::cli::Opts<Parser,Runner,Writer,Custom>::parsed
             at /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/cucumber-0.15.0/src/cli.rs:151:9
```

I've created [an example repo](https://github.com/serzhshakur/yace) with multiple outputs setup where this issue can be observed.

**Comment by tyranron** [comment]
@serzhshakur fixed in #232, released in [0.15.1](https://github.com/cucumber-rs/cucumber/releases/tag/v0.15.1).

**Comment by serzhshakur** [comment]
Oh, that was really fast! Thank you very much indeed!
